posts/tests/test_posts_views.py

```diff
@@ -21,3 +21,25 @@ def test_poll_detail_view(self, client, persist_poll):
         poll_detail_path = f'/post/{persist_poll.pk}/'
         poll_detail_response = client.get(poll_detail_path)
         assert poll_detail_response.status_code == 200
+
+
+@pytest.mark.django_db()
+class TestPostViews:
+    def test_post_detail_view(self, client, persist_resume, persist_poll):
```

Comment:
I think you can delete this function, we are already testing `resume_detail_response.status_code`, `poll_detail_response.status_code`

Reply:
You are right, deleting it now 👍